tests/conftest.py

```diff
@@ -88,6 +94,7 @@ def base_app(request):
         DEBUG=False,
         SECRET_KEY='TEST',
         SECURITY_DEPRECATED_PASSWORD_SCHEMES=[],
+        SQLALCHEMY_TRACK_MODIFICATIONS=False,
```

Comment:
I should note that this isn't part of Globus Auth, but tracking modifications in tests isn't needed and setting this variable prevents a large number of warnings produced every time a test is run.